vendor/engines/c2po/app/views/facility_accounts/account_fields/_purchase_order_account.html.haml

```diff
@@ -14,3 +14,8 @@
   label: text("outside_contact_label"),
   hint: text("outside_contact_hint"),
   hint_html: { :class => 'help-inline' }
+
+= f.input :ar_number,
+  label: text("ar_number_label"),
+  hint: text("ar_number_hint"),
+  hint_html: { :class => 'help-inline' }
```

Comment:
Huh, I thought it would be enforcing the new style hash syntax and double quotes. I'll go poke at the config at some point maybe.